opencti-platform/opencti-front/src/private/components/arsenal/vulnerabilities/VulnerabilityEditionDetails.jsx

```diff
@@ -101,36 +92,33 @@ const VulnerabilityEditionDetailsComponent = (props) => {
 
   const handleSubmitField = (name, value) => {
     if (!enableReferences) {
-      vulnerabilityValidation()
+      vulnerabilityValidator
         .validateAt(name, { [name]: value })
         .then(() => {
-          commitMutation({
-            mutation: vulnerabilityMutationFieldPatch,
+          editor.fieldPatch({
             variables: {
               id: vulnerability.id,
-              input: { key: name, value: value || '' },
+              input: [{ key: name, value: value ?? '' }],
```

Comment:
setting to an empty string won't work if you want to change the field to null.
Especially with numeric fields, the backend will try to parse this empty field and throw an error.

To actually set a nullable field to null (like the score here), you need to do this:
```suggestion
              input: [{ key: name, value: value ?? [null] }],
```

Note the use of an array for value. It's the type expected by GQL, but using a single value works because of type coercion (the little magic in gql.. from one single value it turns it into an array with this single value automatically).

Reply:
nice trick, it works

Reply:
Done ! It works perfectly thank you !